src/fhir/sdc-import-common.js

```diff
@@ -885,7 +885,24 @@ function addCommonSDCImportFns(ns) {
    * @param qr a QuestionnaireResponse instance
    * @returns {{}} an updated LForms form definition, with answer data
    */
-  qrImport.mergeQuestionnaireResponseToLForms = function(formData, qr) {
+  let mismatchItem = [];
+
+  qrImport.mergeQuestionnaireResponseToLForms = function (formData, qr) {
+    let itemsInBoth = qr.item[0].item.filter((obj1) => {
+      let matched = formData.items[0].items.some((obj2) => {
+        return obj1.linkId === obj2.linkId;
+      });
+
+      if (!matched) {
+        mismatchItem.push(obj1.linkId);
+      }
+      return matched;
+      //Handle the missmatch between questionnaire and questionnaireResponse 
+      //Keep only those item in qr whoes present in  formData
+    });
+
+    console.log('Mismatched items linkId:', mismatchItem);
```

Comment:
we should not print unnecessary log message when there are no mismatched items.

Reply:
Okay @semicolin  will handle this situation